code/modules/mob/living/carbon/alien/humanoid/queen.dm

```diff
@@ -16,13 +25,28 @@
 /mob/living/carbon/alien/humanoid/royal/can_inject()
 	return 0
 
+/**
+ * # Alien Queen
+ *
+ * The most important alien caste, the queen is the only reproductive member of the set.
+ *
+ * A subtype of alien which acts as the sole reproductive member of the alien castes.  The queen
+ * boasts the most health and damage of all the xeno types, but is also the slowest.  She can
+ * lay eggs which will create facehuggers, promote other xenos to praetorians, and also call_shuttle
+ * the emergency shuttle after 15 minutes of being alive.
+ */
 /mob/living/carbon/alien/humanoid/royal/queen
 	name = "alien queen"
 	caste = "q"
 	maxHealth = 400
 	health = 400
 	icon_state = "alienq"
+	melee_damage_lower = 20
+	melee_damage_upper = 35
```

Comment:
update this or your changelog


also why not just do 50-50 like TG wanted to do?

Reply:
A guy cried in comments that it will be to much